pom.xml

```diff
@@ -496,6 +491,10 @@
                 <goal>check</goal>
               </goals>
               <configuration>
+                <excludes>
+                  <exclude>io/dapr/durabletask/**/*</exclude>
+                  <exclude>io/dapr/springboot/examples/**/*</exclude>
```

Comment:
Why exclude springboot examples in the durabletask migration PR?

Reply:
@cicoyle Durable tasks was the the first exclusion that we knew we needed to do, hence when I added durable tasks, we can have this low hanging fruit to improve test coverage numbers that really matter.